jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -36,35 +36,26 @@ see the [Configuring CircleCI]({{ site.baseurl }}/2.0/configuration-reference/#p
 
 ## Using the CircleCI CLI to Split Tests
 
-CircleCI supports automatic test allocation
-across your containers.
-The allocation is file-based.
-It requires the CircleCI CLI,
-which is automatically injected
-into your build at run-time.
+CircleCI supports automatic test allocation across your containers. The allocation is file-based. It requires the CircleCI CLI, which is automatically injected into your build at run-time.
```

Comment:
I'm not sure if the sentence "The allocation is file-based" is needed her - does this just mean it's full test files that get allocated across containers/machines rather than partial test files?

Reply:
Full test "thingies" (files, classnames) are split. Usually test runners will accept a list of thingies to run as command-line arguments.
`circleci tests split` helps to assemble these command-line arguments, that's really all it's doing.

So if the test-runner wants to be passed a list of files then the customer needs to give `circleci tests split` a list of files, if the test-runner wants to be passed a list of classnames the customer needs to give `circleci tests split` a list of classnames.

Reply:
Thanks!